shop/src/main/java/ro/msg/learning/shop/service/implementation/OrderServiceImplementation.java

```diff
@@ -0,0 +1,45 @@
+package ro.msg.learning.shop.service.implementation;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.converter.OrderConvertor;
+import ro.msg.learning.shop.dto.OrderDto;
+import ro.msg.learning.shop.repository.OrderRepository;
+import ro.msg.learning.shop.service.interfaces.IOrder;
+import ro.msg.learning.shop.stategy.interfaces.IStrategy;
+
+import java.util.List;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Service
+public class OrderServiceImplementation implements IOrder {
+
+    private final OrderRepository orderRepository;
+    private final OrderConvertor orderConvertor;
+
+    @Override
+    public List<OrderDto> getAllOrders() {
+        return null;
```

Comment:
I hope this is temporary code :D If yes, please request the review on a pull request when you have a complete functionality (I mean without code like this). We should merge in master just complete functionalities.

Reply:
Implemented now